wpilibNewCommands/src/main/native/include/frc2/command/DeferredCommand.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <memory>
+#include <span>
+
+#include <wpi/FunctionExtras.h>
+
+#include "frc2/command/Command.h"
+#include "frc2/command/CommandHelper.h"
+#include "frc2/command/SetUtilities.h"
```

Comment:
What requires this header here?

Reply:
I copied the bulk of the class from ProxyCommand... that header is included in multiple command headers but SetInsert is only used by ScheduleCommand. Fixing. See https://github.com/wpilibsuite/allwpilib/pull/5568